app/assets/stylesheets/building_blocks.scss

```diff
@@ -0,0 +1,83 @@
+html,
+body {
+  background: #FAFAFA;
+}
+
+.card {
+  background: white;
+  border-radius: 2px;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
+  color: #272727;
```

Comment:
Color literals like `#272727` should only be used in variable declarations; they should be referred to via variable everywhere else.